### PR TITLE
[MERGE] GE7: Motorcyle Registry Manufacturing Enhancements

### PR DESCRIPTION
**Pull request by vmac-odoo**
Features:
- This module integrates sales/mrp to motorcycle registry

### CODE REVIEW

File: ge07_team02/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (ayhp-odoo) [inline ge07_team02/__init__.py:1]: New line at the end of the file and all the other files where are missing

File: ge07_team02/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from . import stock_lot
```
Comment (ayhp-odoo) [inline ge07_team02/models/__init__.py:1]: Sort imports in aphabetical order

File: ge07_team02/models/motorcycle_registry.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields, api
```
Comment (ayhp-odoo) [inline ge07_team02/models/motorcycle_registry.py:1]: The imports should be sorted in alphabetical order and check the Odoo guidelines the correct order to sort the import, first go the python libraries then goes the imports from odoo and finally the imports of odoo addons

https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html?highlight=guidelines#imports

File: ge07_team02/models/motorcycle_registry.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        for registry in self.filtered(lambda r: r.vin):
+            match = re.match(pattern, registry.vin)
+            if not match:
+                raise ValidationError('Odoopsie! Invalid VIN')
+            if not registry.vin[0:2] == 'KA':
+                raise ValidationError('Odoopsie! Only motorcycles from Kauil Motors are allowed')
+
+    @api.constrains('stock_lot_ids')
+    def _check_stock_lot_ids(self):
+        for record in self:
```
Comment (ayhp-odoo) [inline ge07_team02/models/motorcycle_registry.py:32]: Use a more descriptive word to name the current value of the loop as you did in the for above

File: ge07_team02/models/stock_lot.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
```
Comment (ayhp-odoo) [inline ge07_team02/models/stock_lot.py:3]: Two lines should be betwwen the imports and the declaration of the class

File: ge07_team02/views/motorcycle_registry_views_inherit.xml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="motorcycle_registry_template_product_form_inherit_ge07" model="ir.ui.view">
+        <field name="name">motorcycle.registry.view.form.inherit.ge07</field>
+        <field name="model">motorcycle.registry</field>
+        <field name="inherit_id" ref="motorcycle_registry.motorcycle_registry_view_form"/>
+        <field name="arch" type="xml">
+
```
Comment (ayhp-odoo) [inline ge07_team02/views/motorcycle_registry_views_inherit.xml:8]: Remove unnecessary blank spaces

File: ge07_team02/views/motorcycle_registry_views_inherit.xml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+            <xpath expr="//page" position="after">
+                <page attrs="{'invisible': ''}" string="Motorcycle Tracking Info">
```
Comment (ayhp-odoo) [inline ge07_team02/views/motorcycle_registry_views_inherit.xml:10]: why invisible is empty?

**Review by ayhp-odoo — CHANGES_REQUESTED** [review]

File: ge07_team02/__init__.py (hunk 1 above)
Comment (vmac-odoo) [reply ge07_team02/__init__.py]: Fixed

File: ge07_team02/models/__init__.py (hunk 2 above)
Comment (vmac-odoo) [reply ge07_team02/models/__init__.py]: Fixed

File: ge07_team02/models/motorcycle_registry.py (hunk 4 above)
Comment (vmac-odoo) [reply ge07_team02/models/motorcycle_registry.py:30]: Fixed

File: ge07_team02/views/motorcycle_registry_views_inherit.xml (hunk 6 above)
Comment (vmac-odoo) [reply ge07_team02/views/motorcycle_registry_views_inherit.xml]: Fixed

File: ge07_team02/views/motorcycle_registry_views_inherit.xml (hunk 7 above)
Comment (vmac-odoo) [reply ge07_team02/views/motorcycle_registry_views_inherit.xml]: Fixed

**Review by ayhp-odoo — APPROVED** [review]

**Pull request merged**